Lombiq.VueJs.Tests.UI/Extensions/BrowserConfigurationExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using Lombiq.Tests.UI.Models;
+using Lombiq.Tests.UI.Services;
+using Lombiq.VueJs.Tests.UI.Assets.Media;
+
+namespace Lombiq.VueJs.Tests.UI.Extensions;
+
+public static class BrowserConfigurationExtensions
+{
+    public static void ConfigureFakeVideoSourceForNegativeTest(this BrowserConfiguration browserConfiguration) =>
+        browserConfiguration.ConfigureFakeVideoSource($"richard.roe.mjpeg");
+
+    public static void ConfigureFakeVideoSourceForPositiveTest(this BrowserConfiguration browserConfiguration) =>
+        browserConfiguration.ConfigureFakeVideoSource($"john.doe.mjpeg");
```

Comment:
No need for string interpolation:
```suggestion
        browserConfiguration.ConfigureFakeVideoSource("richard.roe.mjpeg");

    public static void ConfigureFakeVideoSourceForPositiveTest(this BrowserConfiguration browserConfiguration) =>
        browserConfiguration.ConfigureFakeVideoSource("john.doe.mjpeg");
```